bom/build.gradle

```diff
@@ -81,4 +81,4 @@ publishing {
             }
         }
     }
-}
```

Comment:
Please remove this change

Reply:
I will try. IDEA just keeping updating files which don't have a newline at the end of the file